flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/ScalarFunction.scala

```diff
@@ -58,6 +60,11 @@ abstract class ScalarFunction extends UserDefinedFunction {
 
   override def toString: String = getClass.getCanonicalName
 
+  final def functionIdentifier: String = {
```

Comment:
Can the `functionIdentifier` to be the default implementation of UserDefinedFunction ?  I think it also works for AggregateFunction in the future.